app/models/pydantic/jobs.py

```diff
@@ -134,6 +137,19 @@ class PixETLJob(Job):
     attempt_duration_seconds = int(DEFAULT_JOB_DURATION * 1.5)
 
 
+class GDALCOGJob(Job):
+    """Use for raster transformations using GDAL Python docker in PixETL
```

Comment:
I think better described as:

"""Use for creating COG files using GDAL Python docker in PixETL queue"""